src/common/root/index.js

```diff
@@ -1,12 +1,15 @@
 /* global fetch:false */
 /* global window:true */
-import React, { useState, useEffect } from 'react';
-import Linter from '../linter';
+import 'whatwg-fetch';
```

Comment:
Do we still need it?

Reply:
Probably not. I made only the changes needed to get the demo working with latest dependencies. I've held off making other changes as we'll eventually retire this demo in favour of a client-side one.